openmdao_book/features/building_blocks/surrogates/nearestneighbor.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<p>This is in markdown and missing the proper tags</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Fixed